pom.xml

```diff
@@ -22,6 +22,7 @@
     <javadoc.exec.goal>javadoc-no-fork</javadoc.exec.goal> <!-- stop initialize phase plugins executing twice -->
     <node.version>6.4.0</node.version>
     <npm.version>3.10.3</npm.version>
+    <yarn.version>0.23.0</yarn.version>
```

Comment:
Just to be explicit about the version that `blueocean-plugin` uses, in case the `plugin-pom` gets ticked forward down the road.